string class_name
float64 score
Box box
Mask mask
